AFQ/api.py

```diff
@@ -226,20 +235,16 @@ def __init__(self,
             [BUNDLES] List of scalars to use.
             Can be any of: "dti_fa", "dti_md", "dki_fa", "dki_md"
             Default: ["dti_fa", "dti_md"]
-        wm_criterion : list or float, optional
-            [REGISTRATION] This is either a list of the labels of the white
-            matter in the segmentation file or (if a dictionary is provided)
-            a lower bound and upper bound containting a series of
-            scalar / threshold pairs used for creating the white-matter mask.
-            In the latter case, scalars can be "dti_fa", "dti_md", "dki_fa",
-            "dki_md", thresholds are floats, and 'lb' / 'ub' mean lower and
-            upper bounds respectively (see the format in the default). Each
-            scalar mask will 'logical and' to make the white-matter mask.
-            For an example of the former case,
-            the white matter values for the segmentation provided
-            with the HCP data including labels for midbrain are:
-            [250, 251, 252, 253, 254, 255, 41, 2, 16, 77].
-            Default: {'lb': {'dti_fa': 0.1}, 'ub': {'dti_md': 0.003}}
+        wm_mask : mask from AFQ.mask, optional
+            [REGISTRATION] This is the mask used to make
+            the white matter mask. Currently it is not used for anything.
```

Comment:
We don't use the variable wm_mask for anything (it could be removed, but you said it could useful for PFT?). 
The brain_mask, seed_mask, and stop_mask are all specified with their own mask.